pkg/ingester/limiter.go

```diff
@@ -67,22 +68,22 @@ func NewLimiter(
 
 // AssertMaxSeriesPerMetric limit has not been reached compared to the current
 // number of series in input and returns an error if so.
-func (l *Limiter) AssertMaxSeriesPerMetric(userID string, series int) error {
+func (l *Limiter) AssertMaxSeriesPerMetric(userID string, series int, metric string) error {
 	if actualLimit := l.maxSeriesPerMetric(userID); series < actualLimit {
 		return nil
 	}
 
-	return errMaxSeriesPerMetricLimitExceeded
+	return errors.Wrap(errMaxSeriesPerMetricLimitExceeded, "{metric name: " + metric + "}")
```

Comment:
Can we change `errMaxSeriesPerMetricLimitExceeded` to take metric name as a parameter and we use `errors.Wrapf(errMaxSeriesPerMetricLimitExceeded, metric)`?

Reply:
We already have the labels on the `handleAppendFailure`, no? we can just send those labels down to format the error?

Reply:
Ex: https://github.com/cortexproject/cortex/blob/6be037866e7b71706628b93fe3953129230e548e/pkg/ingester/ingester.go#L1161